README.md

```diff
@@ -51,6 +51,13 @@ To allow the `admin` user from Keycloak to run `oc get sa` run the following:
 
 You should now be able to run `./openshift-api-try` and get a list of service accounts from OpenShift.
 
+## Debugging Mutual TLS
+
+Mutual TLS is sometimes very hard to debug. For this reason we enabled SSL logging on Java. You may also find these two commands helpful
+
+* `openssl s_client -connect 172.17.0.8:8443 -CAfile ./openshift-basedir/kube-apiserver/ca.crt -cert ./openshift-basedir/openshift-apiserver/ca.crt -key ./openshift-basedir/openshift-apiserver/ca.key`
```

Comment:
You are using the CA cert/key as a client cert ??

Reply:
In the early version of this PR, I required a client cert in Keycloak. This was obviously wrong, since I should request a client certificate but do not fail the request, if it's absent. Otherwise I would block whole web console login use case.

Now, the certificate is requested by the Keycloak, but we do not close the connection if it's not there. So I'm thinking what to do with this paragraph. It will probably be the best if I removed it.